src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -340,17 +341,19 @@ class Meta:
             HoofdZaaktypeRelationValidator(),
         ]
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def get_fields(self):
+        fields = super().get_fields()
```

Comment:
refactored because `self.fields` calls `self.get_fields()` and then caches the result (through `@cached_property`). Forcing all fields to be bound in the `__init__` can break certain behaviours and even affect API spec generation.

Reply:
If changing field properties in `Serializer.__init__` is a bad practice should we also change following classes?  

* `BesluitSerializer` 
* `EigenschapSpecificatieSerializer` 
* `InformatieObjectTypeSerializer` 
* `ZaakTypeInformatieObjectTypeSerializer` 
* `BrondatumArchiefprocedureSerializer` 
* `ResultaatTypeSerializer`
* `RolTypeSerializer`
* `ZaakTypenRelatieSerializer`
* `ZaakTypeSerializer` 
* `OndertekeningSerializer`
* `IntegriteitSerializer`
* `EnkelvoudigInformatieObjectSerializer`
* `ObjectInformatieObjectSerializer`
* `RelevanteZaakSerializer`
* `RolSerializer`
* `ZaakObjectSerializer`
* `RolNatuurlijkPersoonSerializer`
* `ObjectInrichtingselementSerializer`
* `ObjectKunstwerkdeelSerializer`
* `ObjectSpoorbaandeelSerializer`
* `ObjectWaterdeelSerializer`

Reply:
Could you add `TODO` comment here about it? So we won't forget which method should be replaced?

Reply:
I've created #1307 for it and assigned it to the next milestone so it's on the board now :)